boards/stm32f429i-disc/Makefile.include

```diff
@@ -14,6 +14,7 @@ include $(RIOTMAKE)/tools/serial.inc.mk
 
 # this board has an on-board ST-link adapter
 export DEBUG_ADAPTER ?= stlink
+export STLINK_VERSION ?= 2-1
```

Comment:
useless, 2-1 is already the default. See [here](https://github.com/RIOT-OS/RIOT/blob/master/makefiles/tools/openocd-adapters/stlink.inc.mk#L3)

Reply:
I know its the default. This was mainly for reference, since it's intended to be changed per board. In other words, by looking at the makefile of the board I see the relevant variables. I can remove it if you feel strongly about this, though.

Reply:
If you look at other boards that uses st-link v2-1, this variable is not used. You could add a comment if you prefer.